### ISSUE TITLE
Various Django admin pages yield 500 errors on AWS-deployed Refinery Instances

### ISSUE THREAD

**Issue by scottx611x** [refinery-platform/refinery-platform#2165]
* Specific code commit: 2a3c825183c423d388c391f52a18e13d5caa2925
* Environment where the error occurred: `test.stemcellcommons.org`

### Steps to reproduce
Please list all the actions and the input data used:

1. Go to the [Django admin UI on the test site](test.stemcellcommons.org/admin)
2. Some links such as: [User Profiles](https://test.stemcellcommons.org/admin/core/userprofile/) & [Tutorials](https://test.stemcellcommons.org/admin/core/tutorials/) yield 500 errors when trying to be viewed.

### Observed behavior
![screen shot 2017-09-18 at 2 52 27 pm](https://user-images.githubusercontent.com/5629547/30558970-08beea92-9c81-11e7-9d6b-cf52b3bf008e.png)
```
[Mon Sep 18 14:50:41.255354 2017] [:error] [pid 31934:tid 139758581556992] /refinery-all-files-test-static/images/tileicon.png
[Mon Sep 18 14:50:41.257810 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675] mod_wsgi (pid=31934): Exception occurred processing WSGI script '/srv/refinery-platform/refinery/config/wsgi_aws.py'.
[Mon Sep 18 14:50:41.257901 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675] Traceback (most recent call last):
[Mon Sep 18 14:50:41.257975 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/wsgi.py", line 187, in __call__
[Mon Sep 18 14:50:41.258062 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     response = self.get_response(request)
[Mon Sep 18 14:50:41.258127 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/base.py", line 199, in get_response
[Mon Sep 18 14:50:41.258185 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     response = self.handle_uncaught_exception(request, resolver, sys.exc_info())
[Mon Sep 18 14:50:41.258230 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/core/handlers/base.py", line 243, in handle_uncaught_exception
[Mon Sep 18 14:50:41.258282 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return callback(request, **param_dict)
[Mon Sep 18 14:50:41.258326 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/utils/decorators.py", line 105, in _wrapped_view
[Mon Sep 18 14:50:41.258378 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     response = view_func(request, *args, **kwargs)
[Mon Sep 18 14:50:41.258422 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/views/defaults.py", line 47, in server_error
[Mon Sep 18 14:50:41.258482 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return http.HttpResponseServerError(template.render(Context({})))
[Mon Sep 18 14:50:41.258528 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 148, in render
[Mon Sep 18 14:50:41.258580 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return self._render(context)
[Mon Sep 18 14:50:41.258624 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
[Mon Sep 18 14:50:41.258674 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return self.nodelist.render(context)
[Mon Sep 18 14:50:41.258719 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 844, in render
[Mon Sep 18 14:50:41.258770 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     bit = self.render_node(node, context)
[Mon Sep 18 14:50:41.258813 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
[Mon Sep 18 14:50:41.258863 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return node.render(context)
[Mon Sep 18 14:50:41.258906 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
[Mon Sep 18 14:50:41.258956 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return compiled_parent._render(context)
[Mon Sep 18 14:50:41.259000 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
[Mon Sep 18 14:50:41.259049 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return self.nodelist.render(context)
[Mon Sep 18 14:50:41.259092 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 844, in render
[Mon Sep 18 14:50:41.259142 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     bit = self.render_node(node, context)
[Mon Sep 18 14:50:41.259185 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
[Mon Sep 18 14:50:41.259243 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return node.render(context)
[Mon Sep 18 14:50:41.259288 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
[Mon Sep 18 14:50:41.259365 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return nodelist.render(context)
[Mon Sep 18 14:50:41.259411 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 844, in render
[Mon Sep 18 14:50:41.259461 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     bit = self.render_node(node, context)
[Mon Sep 18 14:50:41.259504 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
[Mon Sep 18 14:50:41.259561 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return node.render(context)
[Mon Sep 18 14:50:41.259605 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/defaulttags.py", line 312, in render
[Mon Sep 18 14:50:41.259655 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return nodelist.render(context)
[Mon Sep 18 14:50:41.259698 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 844, in render
[Mon Sep 18 14:50:41.259748 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     bit = self.render_node(node, context)
[Mon Sep 18 14:50:41.259791 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
[Mon Sep 18 14:50:41.259842 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return node.render(context)
[Mon Sep 18 14:50:41.259885 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/templatetags/static.py", line 106, in render
[Mon Sep 18 14:50:41.259937 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     url = self.url(context)
[Mon Sep 18 14:50:41.259980 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/contrib/staticfiles/templatetags/staticfiles.py", line 16, in url
[Mon Sep 18 14:50:41.260032 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return static(path)
[Mon Sep 18 14:50:41.260074 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/django/contrib/staticfiles/templatetags/staticfiles.py", line 9, in static
[Mon Sep 18 14:50:41.260126 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     return staticfiles_storage.url(path)
[Mon Sep 18 14:50:41.260168 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/storages/backends/s3boto3.py", line 556, in url
[Mon Sep 18 14:50:41.260221 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     ExpiresIn=expire)
[Mon Sep 18 14:50:41.260264 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/botocore/signers.py", line 570, in generate_presigned_url
[Mon Sep 18 14:50:41.260315 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     params, operation_model)
[Mon Sep 18 14:50:41.260358 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]   File "/home/ubuntu/.virtualenvs/refinery-platform/lib/python2.7/site-packages/botocore/validate.py", line 291, in serialize_to_request
[Mon Sep 18 14:50:41.260409 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675]     raise ParamValidationError(report=report.generate_report())
[Mon Sep 18 14:50:41.260461 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675] ParamValidationError: Parameter validation failed:
[Mon Sep 18 14:50:41.260495 2017] [:error] [pid 31934:tid 139758581556992] [remote 172.31.61.129:27675] Invalid length for parameter Key, value: 0, valid range: 1-inf
```

### Expected behavior
* No 500 errors. Be able to view these assets

**Comment by scottx611x** [comment]
I can still reproduce this issue as of: 7e8847fb36e4a64352247def0ff60fad095abcef

**Comment by jkmarx** [comment]
Resolved by #2863 
Can access user profile and tutorials now.
